src/runtime/runtime_unix.go

```diff
@@ -37,8 +37,8 @@ const tickMicros = 1
 
 // TODO: Linux/amd64-specific
 type timespec struct {
-	tv_sec  int64
-	tv_nsec int64
+	tv_sec  timeT
+	tv_nsec timeT
```

Comment:
This can be implemented much simpler: `int` is 32-bit on 32-bit systems and 64-bit on 64-bit systems. This matches `gc` and I see no reason to change that behavior.
This is different from C because Go is not tied to C legacy where `int` is usually 32-bit even on 64-bit platforms and you can't rely on any particular behavior.